web/book/src/syntax/expressions-and-operators.md

```diff
@@ -15,21 +15,87 @@ filter circumference > 10 && color != "red"
 
-| Group          | Operators         | Precedence | Associativity |
-| -------------- | ----------------- | ---------- | ------------- |
-| identifier dot | `.`               | 1          |               |
-| unary          | `- + ! ==`        | 2          |               |
-| range          | `..`              | 3          |               |
-| mul            | `* / %`           | 4          | left-to-right |
-| add            | `+ -`             | 5          | left-to-right |
-| compare        | `== != <= >= < >` | 6          | left-to-right |
-| coalesce       | `??`              | 7          | left-to-right |
-| and            | `&&`              | 8          | left-to-right |
-| or             | <code>\|\|</code> | 9          | left-to-right |
-| function call  |                   | 10         |               |
+|          Group | Operators         | Precedence | Associativity |
+| -------------: | ----------------- | :--------: | :-----------: |
+| identifier dot | `.`               |     1      |               |
+|          unary | `- + ! ==`        |     2      |               |
+|          range | `..`              |     3      |               |
+|            mul | `* / %`           |     4      | left-to-right |
+|            add | `+ -`             |     5      | left-to-right |
+|        compare | `== != <= >= < >` |     6      | left-to-right |
+|       coalesce | `??`              |     7      | left-to-right |
+|            and | `&&`              |     8      | left-to-right |
+|             or | <code>\|\|</code> |     9      | left-to-right |
+|  function call |                   |     10     |               |
+|   _parens (?)_ | `()`              |     ??     |   see below   |
+
+## Parentheses
+
+PRQL uses parentheses `()` for several purposes:
+
+- Parentheses group operands to control the order of evaluation, for example:
+  `((1 + x) * y)`
+
+- Parentheses set off an [inner transform](./inner-transforms.md) for the
+  `group ()` and `window ()` transforms. _Question: Are there any other
+  transforms that use parentheses like this?_
+
+- Parentheses set off a minus sign for a function argument, for example:
+  `add (-1) (-3)`
+
+- Parentheses set off nested function calls that contain a pipe, either the `|`
+  symbol or a new line. “Nested” means within a transform; i.e. not just the
+  main pipeline, for example: `(column-name | in 0..20)`
+
+- Parentheses wrap a function call that is part of a larger expression, for
+  example: `round 0 (sum distance)` _Question: Is it necessary to state "on the
+  right-hand side of an assignment"? Are there counterexamples?_
+
+  _Question: Why not require parentheses around function calls in a list? Why
+  preserve this (confusing) special case?_
+
+- Parentheses are not required for expressions that do not contain function
+  calls, for example: `foo + bar`.
+
+Here's a set of examples of these rules:
+
+```prql no-fmt
```

Comment:
Can I confirm there are no changes to this section?

(It can be easier to split the PRs up so the file move is in a different PR to the text changes. But I recognize that can also add overhead to the process...)

Reply:
The only change to the example is that I changed "it's contains a pipe" to "it contains a pipe"

(Someday I need to find how to regenerate the snapshots of SQL. I currently use `task test-rust`but that sometimes takes 15 minutes in the Dev Container if crates need to be compiled again.)